src/domain/hooks/hooks.repository.spec.ts

```diff
@@ -100,7 +102,10 @@ describe('HooksRepository (Unit)', () => {
       mockQueuesRepository,
       mockConfigurationService,
     );
-    jest.clearAllMocks();
+  });
+
+  afterEach(() => {
```

Comment:
It seems like we can move `useRealTimers` to `afterAll`. Is there a reason we are doing it after each test?

Reply:
Totally agree in this case! We tend to use `after/beforeEach` to assure test independence and avoid setup interferences when it comes to timers. This is a rule of thumb and I applied that to this test file but it's indeed unnecessary.